src/chrome/content/rules/CakeCentral.com.xml

```diff
@@ -0,0 +1,15 @@
+<!--
+	Mixed content:
+		cakecentral.com
+		www.cakecentral.com
```

Comment:
They redirect server side to HTTPS. DO you have an example for MCB issues?

Reply:
If I didn't include a test URL, it must have been homepage but it looks like it has been fixed now.